repos/c2sm/packages/icon-exclaim/package.py

```diff
@@ -0,0 +1,122 @@
+from spack.pkg.c2sm.icon import Icon as IconC2sm
+
+import os
+import re
+from collections import defaultdict
+import spack.error as error
+
+
+def validate_variant_dsl(pkg, name, value):
+    set_mutual_excl = set(['substitute', 'verify', 'serialize'])
+    set_input_var = set(value)
+    if len(set_mutual_excl.intersection(set_input_var)) > 1:
+        raise error.SpecError(
+            'Cannot have more than one of (substitute, verify, serialize) in the same build'
+        )
+
+
+class IconExclaim(IconC2sm):
+    git = 'git@gitlab.dkrz.de:icon/icon.git'
+
+    maintainers('jonasjucker', 'huppd')
+
+    version('exclaim-master',
+            branch='master',
+            git='git@github.com:C2SM/icon-exclaim.git',
+            submodules=True)
+    version('exclaim',
+            branch='icon-dsl',
+            git='git@github.com:C2SM/icon-exclaim.git',
+            submodules=True)
```

Comment:
```suggestion
    git = 'git@gitlab.dkrz.de:icon/icon-exclaim.git'

    maintainers('jonasjucker', 'huppd')

    version('exclaim-master',
            branch='master',
            submodules=True)
    version('exclaim',
            branch='icon-dsl',
            submodules=True)
```
Also consider renaming the versions.

Reply:
fixed, but differently